enterprise/server/remote_execution/filecache/filecache.go

```diff
@@ -261,6 +258,15 @@ func (c *fileCache) addFileToGroup(groupID string, node *repb.FileNode, existing
 	// which is not good.
 	k := groupSpecificKey(groupID, node)
 
+	sizeOnDisk := node.GetDigest().GetSizeBytes()
+	info, err := os.Stat(existingFilePath)
+	if err != nil {
+		return status.WrapError(err, "stat")
+	}
+	// Stat always returns number of 512 byte blocks, regardless of the FS
+	// settings.
+	sizeOnDisk = info.Sys().(*syscall.Stat_t).Blocks * 512
```

Comment:
I'm guessing this doesn't work on Windows. I'm not sure what the status of Windows executors are, but we may need to special case this calculation for this to compile on Windows @sluongng

Reply:
Windows executor should be functional on HEAD. If you guys need to test something, please ping me on Slack.